backend/src/nodes/nodes/ncnn/upscale_image.py

```diff
@@ -94,15 +103,14 @@ def upscale(
                 )
         except ValueError as e:
             raise e
+        except RuntimeError as e:
+            raise e
```

Comment:
I think just `raise` would be better here.

```suggestion
        except (RuntimeError, ValueError):
            raise
```

Reply:
didn't know you could do that